plugins/inputs/bond/bond_test.go

```diff
@@ -108,7 +108,7 @@ Partner Churned Count: 0
 Slave Interface: eth1
 MII Status: down
 Speed: Unknown
-Duplex: Unkown
+Duplex: Unknown
```

Comment:
This is sample date that comes from a command. Are you sure this command is also producing the typo?

Reply:
> Are you sure this command is also producing the typo?

No. Reverted the change in order to avoid a regression.